README.md

```diff
@@ -86,15 +87,14 @@ int main(void) {
     printf("Uncompressed data length: %zu\n", uncompressed_values.len);
 
     // Configuration for compression.
-    // The C API mirrors the `Method` enum from TerseTS.
-    struct Configuration config = { SwingFilter, 0.1f };
+     const char *cfg = "{\"abs_error_bound\": 0.01}";
 
     // Output buffers for compressed/decompressed data.
     struct CompressedValues   compressed_values = {0};
     struct UncompressedValues decompressed_values = {0};
 
-    // Compression.
-    int32_t result = compress(uncompressed_values, &compressed_values, config);
+    // Compression. The C API mirrors the `Method` enum from TerseTS.
+    int32_t result = compress(uncompressed_values, &compressed_values, SwingFilter, cfg);
```

Comment:
Why is the configuration named `cfg` here instead of the more informative `configuration` as in the `Python` example? I see no upside to use `cfg` instead of `configuration` so I suggest using `configuration in both examples. No matter what version is used, they should be consistent.